src/App.js

```diff
@@ -2,12 +2,65 @@ import React from "react";
 import "./App.css";
 
 class App extends React.Component {
+  constructor() {
+    super()
+
+    this.state = {
+      count: 0,
+      incrementor: 1,
+    }
+  }
+
+  
+  increaseCount = () => {
+    const { count, incrementor } = this.state
+    
+    if(count >= 10) {
+      this.setState ({
+        count: count - 10,
+        incrementor: incrementor + 1,
+      })
+    } else {
+      window.alert("You can't afford that!")
+    }
+  }
+  
+  addOne = () => {
+    const { count, incrementor } = this.state
+
+    this.setState({
+      count: count + incrementor
+    })
+  }
+  
+  resetCount = () => {
+    this.setState ({
+      count: 0,
+      incrementor: 1,
+    })
+  }
+  
   render() {
+    const { count, incrementor} = this.state
+
+    if (count < 100) {
+        return (
+        <main>
+          <h2>Current Score: {count}</h2>
+          <button type="submit" onClick={this.addOne} style={count >= 100 ? {display:"none"} : null }> +{incrementor} </button>
```

Comment:
```suggestion
          <button type="submit" onClick={this.addOne} }> +{incrementor} </button>
```
Since you're already using conditional logic on line 46 to determine what to render to the page, you don't need to use a ternary operator to hide this button if the count >= 100. It's not hurting the functionality of your app, but it's redundant. 

Same thing goes for the button on line 52.

Great job btw!